agent/src/main/java/com/teamscale/jacoco/agent/upload/teamscale/TeamscaleUploader.java

```diff
@@ -33,21 +41,51 @@ public void upload(CoverageFile coverageFile) {
 			if (tryUploading(coverageFile)) {
 				deleteCoverageFile(coverageFile);
 			} else {
-				logger.warn("Failed to upload coverage to Teamscale. " +
-								"Won't delete local file {} so you can upload it yourself manually.",
-						coverageFile);
+				logger.warn("Failed to upload coverage to Teamscale. "
+						+ "Won't delete local file {} so that the upload can automatically be retried upon profiler restart."
+						+ " Upload can also be retried manually.", coverageFile);
```

Comment:
Please consistently place the leading space either in the previous or the next line